examples/make_example/test/test_foo.c

```diff
@@ -13,5 +15,12 @@ void test_foo_init_should_initialize_multiplier()
 {
     foo_init();
 
-    TEST_ASSERT_FALSE(1);
+    CMOCK_MEM_INDEX_TYPE ndx = CMock_Guts_MemNew(32768);
+    void *ptr = CMock_Guts_GetAddressFor(ndx);
+    printf("ndx: %u, ptr: %p, \n", ndx, ptr);
+    printf("free: %u, used: %u, capacity: %u, \n",
+           CMock_Guts_MemBytesFree(), CMock_Guts_MemBytesUsed(), CMock_Guts_MemBytesCapacity());
+
+    TEST_ASSERT_NOT_NULL(ptr);
+    TEST_ASSERT_EQUAL(CMock_Guts_MemBytesFree() + CMock_Guts_MemBytesUsed(), CMock_Guts_MemBytesCapacity());
```

Comment:
This TEST_ASSERT_EQUAL should probably be TEST_ASSERT_LESS_OR_EQUAL. It's okay if we leave extra unused data... in fact, that is the point of the original change... we are making sure that it works out for alignment issues.